sulong/projects/com.oracle.truffle.llvm.parser/src/com/oracle/truffle/llvm/parser/LLVMParser.java

```diff
@@ -126,6 +128,10 @@ private void defineFunction(FunctionSymbol functionSymbol, ModelModule model, Da
         LLVMFunction llvmFunction = LLVMFunction.create(functionSymbol.getName(), library, function, functionSymbol.getType(), runtime.getBitcodeID(), functionSymbol.getIndex(),
                         functionDefinition.isExported());
         runtime.getFileScope().register(llvmFunction);
+        final boolean cxxInterop = LLVMLanguage.getContext().getEnv().getOptions().get(SulongEngineOption.CXX_INTEROP);
```

Comment:
You can use `parserRuntime.getContext()` here, that's more efficient. `LLVMLanguage.getContext()` needs to look up the context from a thread-local.